ci-operator/step-registry/openshift/e2e/test/pausedworkers/openshift-e2e-test-pausedworkers-ref.yaml

```diff
@@ -0,0 +1,64 @@
+ref:
+  as: openshift-e2e-test-pausedworkers
+  from: tests
+  commands: openshift-e2e-test-pausedworkers-commands.sh
+  timeout: 10800s
+  env:
+  - name: TEST_TYPE
+    default: "suite"
+    documentation: |
+      The type of test to perform.
+
+      * 'suite' - Run the test suite defined by TEST_SUITE. By default this executes the
```

Comment:
I think we would also need to run serial suite

Reply:
Test suite (like conformance or serial) is set in `TEST_SUITE`, this var is used to determine the test sequence - run TEST_SUITE / run TEST SUITE and conformance (for disruption for instance ) / upgrade with no tests / upgrade and run TEST_SUITE